lib/Screens/statistics.dart

```diff
@@ -1,11 +1,23 @@
+import 'package:flutter/cupertino.dart';
```

Comment:
this is a really long class, think about exporting some functionality for example to a statistics helper class

Reply:
+1